std/js/Browser.hx

```diff
@@ -26,6 +26,12 @@ import js.html.Storage;
 import js.html.XMLHttpRequest;
 
 class Browser {
+	/** The global scope with additional fields available only in a worker context. */
+	public static var self(get, never):js.html.WorkerGlobalScope;
+
+	extern inline static function get_self():js.html.WorkerGlobalScope
+		return untyped __js__("self");
```

Comment:
Please no more `untyped __js__` in new additions. There's `js.Sytnax.code` and such to deal with this.